robber/expect.py

```diff
@@ -32,14 +33,14 @@ def remove_custom_message(cls):
         cls.message = None
 
     @classmethod
-    def register(cls, name, klass):
+    def register(cls, name, klass, is_negative=False):
         cls.matchers[name] = klass
 
         def method(self, other=None, *args):
-            return klass(self.object, other, *args).fail_with(self.message).match()
+            fact = is_negative is not self.is_negative
```

Comment:
Hm, I rather write a FalseMatcher to make it cleaner than re-use the TrueMatcher in this case. It seems magical to me. Anyway, do it your choice then.